app/assets/javascripts/bastion/components/global-notification.service.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * @ngdoc service
+ * @name  Bastion.components.service:GlobalNotification
+ *
+ * @description
+ *  Service to display a global notifcation
+ */
+angular.module('Bastion.components').service("GlobalNotification", [
+    function () {
+        this.errorMessages = [];
+        this.successMessages = [];
+
+        this.setSuccessMessage = function (message) {
+            this.successMessages.push(message);
```

Comment:
That is something I noticed as well, I think its worth having a discussion. I am in favor of not having to manually exit out of the notifications, I think they should time out, though maybe some users like having the errors persist?

One thought I have is to default the messages to time out, but have an argument available to persist the notification. The idea being something like "Product X successfully created" would time out, and more important notifications like  "Error creating Product X: [error message and info]" would stay up until the user exited out.

Either way, I think this can be added outside of this PR if we decide a change is necessary :)